class-job.php

```diff
@@ -51,6 +51,8 @@ public function save() {
 			$data['interval'] = $this->interval;
 		}
 
+		wp_cache_delete( "jobs:{$this->site}", 'cavalcade-jobs' );
```

Comment:
In fact, the object cache won't be loaded at all by the Runner, so this needs to check if the cache functions exist. We may actually be better off completely ignoring the object cache and instead storing this in an internal variable cache based on that, hmm...